build_tools/setup_helpers/extension.py

```diff
@@ -32,6 +32,7 @@ def _get_build(var):
 
 
 _BUILD_SOX = _get_build("BUILD_SOX")
+_BUILD_KALDI = _get_build("BUILD_KALDI")
```

Comment:
Because we currently default to build Kaldi, in case the environment variable is missing, we need to default to `true` for this.

Please update the `_get_build` to allow override of default value.